joomla/joomla-cms
# New Feature: add redirect option after article submission

Replacing https://github.com/joomla/joomla-cms/pull/13011 after discussing with @bembelimen 

**Testing Instructions**

=> Create a frontend `Create Article` menu item
=> Look in the tab "Options"
=> The only parameter is to choose or not a specific category
=> Submit an article in the frontend, you'll redirected to the front page
=> Apply patch
=> Look into the menu item options again, there should be a new parameter `Submission Redirect`

![screen shot 2016-12-17 at 11 00 10](https://cloud.githubusercontent.com/assets/869724/21285976/0ea37e30-c448-11e6-97ae-01dd83ed4e84.png)

=> Do not Select any menu item and submit a new article, the redirect to the frontpage should still work
=> Choose a new redirect target in the menu item option
=> Submit an article
=> The user should be redirected to the new (defined) target

Now works in multilang and only when creating an article when the menu item exists.

Thanks @bembelimen 

Note: also corrected some missing JRoute for redirects in the same file.
